src/test/java/jp/ikedam/jenkins/plugins/updatesitesmanager/ManagedUpdateSiteJenkinsTest.java

```diff
@@ -26,7 +26,6 @@
 import com.tngtech.java.junit.dataprovider.DataProvider;
 import com.tngtech.java.junit.dataprovider.DataProviderRunner;
 import hudson.util.FormValidation;
-import jenkins.model.DownloadSettings;

```

Comment:
Removed in Jenkins-2.200: https://github.com/jenkinsci/jenkins/pull/3970/files#diff-d8406ff5c3050cfdc73460ad451d2f720548551f843149079df7e5719bbfcec5